lib/internal/http2/compat.js

```diff
@@ -417,7 +417,11 @@ class Http2ServerResponse extends Stream {
   write(chunk, encoding, cb) {
     var stream = this[kStream];
     if (stream === undefined) {
-      cb(new Error('HTTP/2 Stream has been closed'));
+      var err = new Error('HTTP/2 Stream has been closed');
+      if (cb)
```

Comment:
So I opened a PR with a change like that in core recently. The pushback was that it introduces silent failure instead of throwing when you try to call a truthy value that isn't actually a function.